tomviz/python/AutoCrossCorrelationTiltImageAlignment.py

```diff
@@ -68,6 +68,10 @@ def transform_scalars(self, dataset):
 
         # Create a spreadsheet data set from table data
         column_names = ["X Offset", "Y Offset"]
+        offsets[np.where(offsets > tiltSeries.shape[0] / 2), :] = 
+            offsets[np.where(offsets > tiltSeries.shape[0] / 2), :] - tiltSeries.shape[0] / 2
```

Comment:
BTW: you can store the result of your `np.where(offsets > tiltSeries.shape[0] / 2)` in a variable if you don't want to repeat the whole thing.  This will help with readability.